vql/server/users/grant.go

```diff
@@ -72,6 +73,13 @@ func (self GrantFunction) Call(
 		return vfilter.Null{}
 	}
 
+	services.LogAudit(ctx,
```

Comment:
Should we only log if successful?

Reply:
Updated the PR to only log successful events.